src/contexts/QwestiveReferral/QwestiveReferral.ts

```diff
@@ -0,0 +1,76 @@
+import { useContext, useEffect, useState } from 'react'
+import {
+  IQwestiveContext,
+  SDKContext,
+  TInitSDK,
+  TQwestiveTracker,
+} from './QwestiveReferralContext'
+
+/**
+ * Methods added on the window object by src/components/common/Head/scripts/QwestiveScripts.tsx
+ */
+declare global {
+  interface Window {
+    QwestiveTracker: {
+      init: ({
+        apiKey,
+        projectId,
+      }: TInitSDK) => Omit<TQwestiveTracker, 'isLoading'>
+      loadQwestiveTracker: (callback: () => void) => void
+      isInitialized: boolean
+    }
+  }
+}
+
+// Juicebox-referral campaign: https://juicebox.referral.qwestive.io/referral/hJCUZVJIodVP6Ki6MP6e
+const QWESTIVE_REFERRAL_PROJECT_ID = 'ma6zYcqmv0hInMskIQf1'
+const QWESTIVE_REFERRAL_API_KEY = 'xgXZwdZMAAY71tcLAdexo2VM091EitpEL1ScpoP/Z4A='
```

Comment:
**question** is it okay to publicly expose this?

Reply:
yes, that is public API Key that we use to initialize the SDK. So, no worries.